packages/vue-country-flag/src/CountryFlag.vue

```diff
@@ -65,12 +65,14 @@ export default {
 </script>
 <style scoped>
   .small-flag {
+    margin: -0.9em -1.2em -0.9em -1.2em;
```

Comment:
Should be
```css
margin: -15px -20px;
```

original is 52x39, remainder of 0.25 scale : 39x29.25, divided by 2 : 19.5x14.6